src/app/views/apps/modals/app-connect-modal-cloud.controller.coffee

```diff
@@ -0,0 +1,57 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller('DashboardAppConnectCloudModalCtrl', ($scope, $uibModalInstance, app, MnoeAppInstances, toastr) ->
```

Comment:
For new modals, please create a component modals. (cf. https://angular-ui.github.io/bootstrap/#!#modal)